src/networking/telemetry.js

```diff
@@ -1 +1 @@
-module.exports = {name: 'react-native-auth0', version: '2.13.3'};
+module.exports = {name: 'react-native-auth0', version: '2.14.0'};
```

Comment:
Should we use `2.14.0-fa.0` here so that we can distinguish between the FA release and actual release in the logs? Normally this value would be the same as what's in `package.json`.